lib.rs

```diff
@@ -272,9 +306,27 @@ fn process_toml(cargo_toml: &str) -> Result<String, String> {
     for (f, top, comment) in features {
         let default = if default_features.contains(f) { " *(enabled by default)*" } else { "" };
         if !comment.trim().is_empty() {
-            writeln!(result, "{}* **`{}`**{} —{}", top, f, default, comment).unwrap();
+            if let Some(feature_label) = feature_label {
+                writeln!(
+                    result,
+                    "{top}* {label}{default} —{comment}",
```

Comment:
I forgot if `"{variable}"` is supported by the minimum supported rust version, but i think it might not be.

Reply:
I'll revert this unnecessary change, sorry for that.

Reply:
Addressed by 6395a1fcecd1a894ae3e580a7afc7f3a82be6891.